modules/migration-migrating-on-prem-to-cloud.adoc

```diff
@@ -29,7 +29,7 @@ A service created on the destination cluster exposes the source cluster's API to
 +
```

Comment:
@pranavgaikwad This file is NOT called by (include:: etc)  the Advanced migration options master file, advanced-migration-options-mtc.adoc, so it does not appear in the MTC guide. Is that OK?

Reply:
Should be fine